### ISSUE TITLE
Cancel does not work where I'm using isContainer

### ISSUE THREAD

**Issue by philiphendry** [bevacqua/dragula#221]
I've written the code below in an Aurelia app where I'm dragging toolbox items into a form where they are rendered differently. I therefore want to cancel the drag so I can handle the rendering. I'm also using `isContainer` because the content doesn't exist at the right time in Aurelia and `copy` because I don't want the toolbox 'widget' to disappear.

However, the `cancel()` in the `drop` event doesn't work. Is this code valid?

    constructor() {
        let toolbox = dragula({
          isContainer: el => {
            return el.classList.contains("drag-source")
              || el.classList.contains("drag-target");
          },
          copy: true,
          revertOnSpill: true
        });

        toolbox.on("drop", (el, container, source) => {
          toolbox.cancel();
        });
      }

For the moment I've replaced `toolbox.cancel()` with `el.remove()` which seems to do the trick but I wonder whether this carries the correct intention.

**Comment by bevacqua** [comment]
It's too late on drop to cancel, see #188 as a duplicate of this one.

**Comment by philiphendry** [comment]
Thanks for the response. I kinda thought it would be too late but I was following some code I found in the [Aurelia Klondike](https://github.com/jdanyow/aurelia-solitaire/blob/master/src/drag-and-drop.js#L41) example.

**Comment by bevacqua** [comment]
Well you could just move the element back to the source.

**Comment by philiphendry** [comment]
Thanks @bevacqua. As mentioned above I've swapped it for 'el.remove()' and that seems to be working. I had lots of other trouble because I was writing ES6 arrow functions in the is* functions and it would appear the babel transpiler messes up logical expression spread over one line (like my 'isContainer' code above.) I spent ages trying to figure out that I just needed to put them all on one continuous. After that Dragula really became an absolute joy to use. 😀
